packages/core/src/modules/credentials/services/CredentialService.ts

```diff
@@ -109,9 +109,9 @@ export class CredentialService {
     })
 
     // Set the metadata
-    credentialRecord.metadata.set('_internal/indyCredential', {
+    credentialRecord.metadata.set(CredentialMetadataKeys.IndyCredential, {
       schemaId: options.schemaId,
-      credentialDefinintionId: options.credentialDefinitionId,
+      credentialDefinitionId: options.credentialDefinitionId,
```

Comment:
Thank god for typing :)

When was this typo introduced? Can it cause problems?

Reply:
This can cause a problem if the metadata is set at the `createProposal` function and was never overwritten. I think the chances of that, and the metadata still being used, is slim to none.